src-electron/generator/template-util.js

```diff
@@ -206,6 +206,8 @@ async function ensureZclPackageIds(context) {
 async function ensureTemplatePackageId(context) {
   if ('templatePackageId' in context.global) {
     return context.global.templatePackageId
+  } else if ('genTemplatePackageId' in context.global) {
```

Comment:
I'm confused by this. Why are we returning one or the other?

Reply:
I am not seeing templatePackageId under context.global. My guess is this somehow got changed to genTemplatePackageId somewhere down the line. So I thought keep `templatePackageId` for backwards compatibility just in case and capture the missing `genTemplatePackageId` here.